src/main/java/com/actionlistapi/model/ActionRequest.java

```diff
@@ -19,140 +19,136 @@
 @Entity
 public class ActionRequest {
 	
-	@Id
-	@GeneratedValue(strategy = GenerationType.IDENTITY,generator="id_seq")
-	@SequenceGenerator(name="id_seq",sequenceName="id_seq",allocationSize=1)
-	private long id;
+	@Id 
+	@GeneratedValue(generator="uuid")
+	@GenericGenerator(name="uuid",strategy = "uuid")
+	private String id;
 	
-	private String processInstanceId;
+    private String processInstanceId;
 	
-	@ManyToOne
-	private ProcessType processType;
+	private String username;
 	
-	private String title;
+	@OneToOne
```

Comment:
Why did this relationship change to one-to-one? I'd think it would still be many-to-one.